tests/unit/test_fetch.py

```diff
@@ -123,7 +123,7 @@ def test_start_service(mocker, tmp_path):
                     "--idle-shutdown=300",
                 ]
             )
-            + f" > {fetch._get_log_filepath()}",
+            + f" > {fetch.get_log_filepath()}",
```

Comment:
But wait, shouldn't it be `--log-file {fetch.get_log_filepath()}` instead of the redirection ?

Reply:
oh snap you're super right! I got my branches confused. I'll drop the reviewers from this until it's ready again, sorry for the noise!